functions/pubsub/main.py

```diff
@@ -22,7 +22,7 @@
 
 # Instantiates a Pub/Sub client
 publisher = pubsub_v1.PublisherClient()
-PROJECT_ID = os.getenv('GCP_PROJECT')
+PROJECT_ID = os.getenv('GOOGLE_CLOUD_PROJECT')
```

Comment:
Do these point to the same thing in the Functions environment? https://cloud.google.com/functions/docs/env-var#nodejs_6_nodejs_8_python_37_and_go_111

Reply:
Good catch - @grayside I assume this is still the right env var to use?

Reply:
`GOOGLE_CLOUD_PROJECT` is the generally correct variable to use and present in some of the older runtimes. It must be user-supplied on deployment for newer runtimes, the current practice is to use the metadata server to get the current projectID.